packages/victory-errorbar/src/index.d.ts

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import {
+  EventPropTypeInterface,
+  StringOrNumberOrCallback,
+  VictoryDatableProps,
+  VictoryCommonProps,
+  VictoryLabelableProps,
+  VictoryMultiLabelableProps,
+  VictoryStyleInterface
+} from "victory-core";
+
+type dataType = {
+  x?: string | number;
+  y?: string | number;
+  errorX?: number | number[];
+  errorY?: number | number[];
+};
+
+export interface VictoryErrorBarProps
+  extends VictoryCommonProps,
+    VictoryDatableProps,
+    VictoryLabelableProps,
+    VictoryMultiLabelableProps {
+  borderWidth?: number;
+  errorX?: number | number[];
```

Comment:
`errorX` and `errorY` are data accessor props like `x` and `y` one other components. They should use the same type as those, which includes functions, strings, and arrays of strings.

Reply:
Ah! Sorry, question: is an array of ints allowed?